src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```diff
@@ -652,3 +652,10 @@ services:
             - '@router.default'
         tags:
             - { name: form.type }
+
+    prestashop.bundle.form.form.admin.sell.customer.required_fields:
```

Comment:
You started using new naming convention 😛 please do it in another PR
so this PR is a "feature" type PR which means it modifies a behavior using known classes/methods
while the other PR will be a "refactoring" type PR: the app (prestashop CMS) behavior is unchanged, but the underlying implementation is modified

PR 1 : QA expects changes
PR 2: QA expects no visual changes